addon/components/nypr-account-forms/signup.js

```diff
@@ -83,7 +93,8 @@ export default Component.extend({
     });
   },
   signUp() {
-    return get(this, 'newUser').save();
+    let newUser = get(this, 'newUser');
+    return newUser.save();
```

Comment:
why this style change?

Reply:
oops, I temporarily had some logic applied to `newUser` here, and then removed it